scripts/sv_logger.lua

```diff
@@ -14,7 +14,7 @@ local function getPlayerData(src)
     if not src then
         return false
     end
-    if src == 0 then return {name = 'console', identifiers = {}} end
+    if src <= 0 then return {name = 'console', identifiers = {}} end
```

Comment:
This would break the loggers check for console. This conditional would never be true as source cannot be less than 0. Shouldn't need to be changed from its original code

Reply:
ok